.github/workflows/scheduled-validation.yml

```
@@ -0,0 +1,303 @@
+          ${{ steps.status.outputs.status == 'failed' && '⚠️ **Action Required:** Review failed tests and create fixes' || '✅ All tests passing - template is healthy' }}
+
+          ---
+
+          [View Full Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
+          EOF
+
+      - name: Create issue on failure
+        if: steps.status.outputs.status == 'failed'
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.9)

got unexpected character '?' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)